compose.yaml

```
@@ -26,10 +26,10 @@ services:
       - /app/public/sites
     restart: always
 
-  postgres:
+  mariadb:
     build:
       context: .
-      dockerfile: db.Dockerfile
+      dockerfile: mariadb.db.Dockerfile
     environment:
-      POSTGRES_PASSWORD: example
+      MARIADB_ROOT_PASSWORD: drupal
```

```suggestion
      MARIADB_ROOT_PASSWORD: example
```

To be consistent with the Postgres example.